builtin/plugins/chainconfig/chainconfig.go

```diff
@@ -13,6 +13,7 @@ import (
 	"github.com/loomnetwork/loomchain/builtin/plugins/dposv3"
 	"github.com/loomnetwork/loomchain/registry"
 	"github.com/pkg/errors"
+	"sort"
```

Comment:
You should be using `goimports` for formatting so it moves imports of built-in modules up top.